<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSet</class>
 <widget class="QWidget" name="ParamSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../imgs.qrc">
     <normaloff>:/new/imgs/images/back1.jpg</normaloff>:/new/imgs/images/back1.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.011, stop:0.405128 rgba(145, 145, 145, 255), stop:0.887179 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#ffffff;&quot;&gt;      终端参数设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mntcom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯地址配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_passage">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯通道配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_meter">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>档案配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pulse">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>脉冲遥信配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_group">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>总加组配置</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>736</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_time">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_password">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>105</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>变更密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../imgs.qrc"/>
 </resources>
 <connections/>
</ui>
